### PR TITLE
Update shader preloader with missing shader usages

### PR DESCRIPTION
**Pull request by peppy**
Noticed stutter on first load of https://github.com/ppy/osu/pull/29936.

### CODE REVIEW

File: osu.Game/Screens/Loader.cs [hunk 1]
```diff
@@ -118,13 +118,20 @@ private void load(ShaderManager manager)
...
+                loadTargets.Add(manager.Load(VertexShaderDescriptor.TEXTURE_2, @"CircularProgress"));
+                loadTargets.Add(manager.Load(VertexShaderDescriptor.TEXTURE_2, @"ArgonBarPath"));
+                loadTargets.Add(manager.Load(VertexShaderDescriptor.TEXTURE_2, @"ArgonBarPathBackground"));
+                loadTargets.Add(manager.Load(VertexShaderDescriptor.TEXTURE_2, @"SaturationSelectorBackground"));
+                loadTargets.Add(manager.Load(VertexShaderDescriptor.TEXTURE_2, @"HueSelectorBackground"));
 
                 loadTargets.Add(manager.Load(VertexShaderDescriptor.TEXTURE_3, FragmentShaderDescriptor.TEXTURE));
+
+                loadTargets.Add(manager.Load(@"CursorTrail", FragmentShaderDescriptor.TEXTURE));
+                loadTargets.Add(manager.Load(@"LogoAnimation", @"LogoAnimation"));
```
Comment (bdach) [inline osu.Game/Screens/Loader.cs:134]: Not sure if oversight or intentionally skipped, but there's also this one:

https://github.com/ppy/osu/blob/5efdc6cdd74cecec6e08087c88aa638973ff52ac/osu.Game.Rulesets.Osu/Skinning/Argon/ArgonSpinnerProgressArc.cs#L101

Comment (peppy) [reply osu.Game/Screens/Loader.cs:134]: Hmm, this is in a ruleset implementation so I'm not 100% sure, but let's add it for now.

Comment (bdach) [reply osu.Game/Screens/Loader.cs]: So is `CursorTrail` though.

File: osu.Game/Screens/Loader.cs [hunk 2]
```diff
@@ -118,13 +118,21 @@ private void load(ShaderManager manager)
             {
                 loadTargets.Add(manager.Load(VertexShaderDescriptor.TEXTURE_2, FragmentShaderDescriptor.TEXTURE));
                 loadTargets.Add(manager.Load(VertexShaderDescriptor.TEXTURE_2, FragmentShaderDescriptor.BLUR));
+                loadTargets.Add(manager.Load(VertexShaderDescriptor.TEXTURE_2, FragmentShaderDescriptor.TEXTURE));
```
Comment (smoogipoo) [inline osu.Game/Screens/Loader.cs:121]: This one looks to be a duplicate

**Review by smoogipoo — CHANGES_REQUESTED** [review]

**Review by smoogipoo — APPROVED** [review]
We really ought to improve this at some point. It's pretty bad to refer to shaders (especially those in external projects) by name...

**Review by bdach — APPROVED** [review]

**Pull request merged**